### PR TITLE
News: GRASS GIS 8.3.1 released

### PR DESCRIPTION
**Pull request by neteler**
News item for the GRASS GIS 8.3.1 release.

image

Note that I have re-used the 8.3.0 screenshot. Please substitute it in this PR if possible.

Update of the relevant files:

- [x] data/grass.json
- [x] content/about/history/releases.md
- [x] content/news/2023_10_25_GRASS_GIS_8_3_1_released.md

### CODE REVIEW

File: content/news/2023_10_25_GRASS_GIS_8_3_1_released.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+compared to 8.3.0. This new patch release brings in
+**important fixes and improvements** in GRASS GIS modules and the
+graphical user interface (GUI) which stabilizes the new
+**single window layout** active by default.
+
+Some of the most relevant changes include: fixes for `r.watershed` which
+got partially broken in the 8.3.0 release; and a fix for installing addons
+on MS Windows with `g.extension`.
+
+Translations have been continued in
```
Comment (veroandreo) [inline content/news/2023_10_25_GRASS_GIS_8_3_1_released.md:20]:
```suggestion
Translations continue in
```

File: content/news/2023_10_25_GRASS_GIS_8_3_1_released.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+[Weblate](https://weblate.osgeo.org/projects/grass-gis/), which
+automatically creates pull requests with the translated chunks.
+We'd like to **thank the translators of all languages** for their
+ongoing support!
+
+<a href="/images/news/grass83_news_screenshot.png">
+  <img src="/images/news/grass83_news_screenshot.png"
```
Comment (veroandreo) [inline content/news/2023_10_25_GRASS_GIS_8_3_1_released.md:27]: What if we use the one that Michael Barton created for #3129: https://www.dropbox.com/scl/fi/v05vth4lutpuaex5sw2uu/GRASS8.3_thematic.png?rlkey=pnq2wus10336hfdn5fftggwww&dl=0

**Review by veroandreo — APPROVED** [review]

File: content/news/2023_10_25_GRASS_GIS_8_3_1_released.md (hunk 2 above)
Comment (neteler) [reply content/news/2023_10_25_GRASS_GIS_8_3_1_released.md:27]: Good idea. Does @cmbarton agree?

**Comment by neteler** [comment]
Thanks, @cmbarton !

Replaced in PR:

image

File: content/news/2023_10_25_GRASS_GIS_8_3_1_released.md (hunk 2 above)
Comment (cmbarton) [reply content/news/2023_10_25_GRASS_GIS_8_3_1_released.md]: I am of course happy to have my image used like this. Glad you all like it.

**Comment by neteler** [comment]
Published at
https://grass.osgeo.org/news/2023_10_25_grass_gis_8_3_1_released

**Pull request merged**